docs/source/guides/snippets/tracing/bentoml_configuration.yaml

```diff
@@ -6,16 +6,16 @@ api_server:
       enabled: true
     max_concurrent_streams: 100
     maximum_concurrent_rpcs: 1
-  tracing:
-    sample_rate: 0.7
-    exporter_type: jaeger
-    jaeger:
-      thrift:
-        agent_host_name: jaeger
-        agent_port: 6831
```

Comment:
Should we move tracing under `api_server` and `runners` to allow separate configuration of tracing?

Reply:
I think we can, but for this one I think we should just update to the latest documentation.
+1 for separate the trace config for runners and api_server